public/static/app/perf/perf.js

```diff
@@ -762,6 +713,7 @@ mciModule.controller('PerfController', function PerfController(
     $http.get(cedarApp + "/rest/v1/perf/task_id/" + $scope.task.id).then(
       function(resp) {
         var formatted = $filter("expandedMetricConverter")(resp.data);
+        $scope.tempPerfSample = formatted;
```

Comment:
Would it be cleaner to add this as a parameter to legacySuccess so that the scope isn't polluted with temp data?


      $http.get("/plugin/json/task/" + $scope.task.id + "/perf/").then(resp=> legacySuccess(formatted, resp),legacyError);

Reply:
done